### DISCUSSION TITLE
Data for notebook 00

### DISCUSSION THREAD

**Discussion by taylor-obrien** [earthlab-education/Earth-Analytics-2023-01-Intro#289] [Q&A03 Python] [answered]
Is the table at the bottom of the page for the data link in notebook 00 the data that we should be using when creating our monthly variables? The data in the table does not match with what is currently in the notebook, so I am a little confused. Attached a picture below to show the data I am talking about.

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/143227722/7d554939-463c-44fe-8621-72f74589ea23)

**Comment by eculler** [comment, marked as answer]
@taylor-obrien Hm,looks like you might be on a different NOAA page. Check out https://www.ncei.noaa.gov/access/monitoring/climate-at-a-glance/ - you should be able to find a button that says `CSV` on the Boulder data page and right-click it to get the link.
